serverctl/iptables.go

```diff
@@ -0,0 +1,130 @@
+package serverctl
+
+import (
+	"errors"
+	"net"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/gravitl/netmaker/logger"
+	"github.com/gravitl/netmaker/netclient/ncutils"
+	"github.com/gravitl/netmaker/servercfg"
+)
+
+const NETMAKER_PROCESS_NAME = "netmaker"
```

Comment:
lacking comment, also if it's only used in this file, should be private, then no comment needed